features/dd-sdk-android-trace/api/dd-sdk-android-trace.api

```diff
@@ -6034,6 +6049,7 @@ public class com/datadog/trace/core/CoreTracer$CoreTracerBuilder {
 
 public class com/datadog/trace/core/DDSpan : com/datadog/trace/api/profiling/TransientProfilingContextHolder, com/datadog/trace/bootstrap/instrumentation/api/AgentSpan, com/datadog/trace/bootstrap/instrumentation/api/AttachableWrapper, com/datadog/trace/core/CoreSpan {
 	public static final field CHECKPOINTED_TAG Ljava/lang/String;
+	public fun <init> (Ljava/lang/String;JLcom/datadog/trace/core/DDSpanContext;Ljava/util/List;)V
```

Comment:
minor: given that we have a method to call private constructor now, should we make this constructor public?

Reply:
actually not...

Reply:
I am not really sure where this is coming from as I am just exposing the class not the constructors so I will leave it as is. I need the class exposed to cast to `DDSpan` in the unit tests.

Reply:
It is from this change https://github.com/DataDog/dd-sdk-android/pull/1924/files#diff-1c12b21f74915bf39d879d5d17374ad221d56bb67bed99743adc8a86dfbf944cR127 where private constructor was made public. But that is ok I believe, just was wondering about the consistency in the approach (since for `PendingTrace` we are using reflection to create an instance).